### PR TITLE
update deploy steps, again 😈

### PR DESCRIPTION
**Pull request by angaither**
#### What's this PR do?
attempt numero dos on running migrations

#### How should this be manually tested?
noy idea

I think that's the pattern here, I have to say after X , then migrate.

### CODE REVIEW

File: config/deploy.rb [hunk 1]
```diff
@@ -51,4 +51,4 @@
 
 after "deploy:update", "deploy:cleanup"
 after "deploy:symlink", "deploy:link_folders"
-after "deploy:artisan_migrate", "deploy:artisan_cache_clear"
+after "deploy:link_folders", "deploy:artisan_migrate"
```
Comment (weerd) [inline config/deploy.rb]: do you need this `"deploy:artisan_cache_clear"`?

Comment (angaither) [inline config/deploy.rb]: I think we added that in the voting app, to fix something, but thinking about it here, would just mean we would wipe user info on every deploy, so we probably don't want that.

**Comment by DFurnes** [comment]
:+1: :gem:

File: config/deploy.rb (hunk 1 above)
Comment (weerd) [inline config/deploy.rb]: Ah, gotcha. Ok. Didn't know it was a vestige from the Voting App!

**Comment by weerd** [comment]
:+1:

File: config/deploy.rb (hunk 1 above)
Comment (DFurnes) [inline config/deploy.rb]: Yeah, it’d be nice to have a “clean” starter Wercker & Cap config files. We still have some strange public folder linking step that I can't seem to remove that tries to do something to a `public/images` and `public/stylesheets` folder on every Northstar deploy. :grimacing:

Comment (angaither) [inline config/deploy.rb]: yeah, happen in gladiator as well & I didn't see _where_ that was happening

Comment (DFurnes) [inline config/deploy.rb]: Yeah, it's weird. Made an issue at DoSomething/devops#80 to track.

**Pull request merged**